signrelease.sh

```diff
@@ -82,7 +85,7 @@ for ext in $ARCHIVE_TYPES; do
     fi
 
     # sign archive
-    gpg --armor --detach-sign "$TMP_DIR/${project}-${tag}.${ext}"
+    gpg -u "${originGitHub##*/} release" --armor --detach-sign "$TMP_DIR/${project}-${tag}.${ext}"
```

Comment:
Thought about this one some more: How about we ask, like we do for other such options and just default it to the `"${project} relese"` form?

Would need to add this around lines 75 or 76:

```
read -rp "Enter the GPG private key name [${project} release]: " gpgPrivKey
gpgPrivKey=${gpgPrivKey:-$project release}
```

And then change line 88 to:

```suggestion
    gpg -u "${gpgPrivKey} release" --armor --detach-sign "$TMP_DIR/${project}-${tag}.${ext}"
```

Let me know if you'd be onboard with this change and I'll update the branch accordingly.

Reply:
Sounds like a good idea!

BTW another idea is to get the key value (and maybe use that as a default) from the setting in `.gitconfig` (can be get via the `git config` command). Assuming this is the same key…

E.g.:
```
$ git config user.signingkey 
ABA9B8F6F448B07FD7EA4A1A05D40A636AFAB34D
```

As this is a common place to set such an option and one can set/overwrite such options in the usual git config hierarchy…

Reply:
I've update the branch with the question and your suggested term `gpgPrivateKey`. Using the git commit key would again not work for PrivateBin, as I use a different personal key for signing commits than the shared maintainer key for the releases. This may be extra effort, but at least the different identities can be separated.